src/components/Drawer/Drawer.style.ts

```diff
@@ -3,9 +3,11 @@ import { rem } from 'polished';
 import { transition } from 'theme/functions';
 import { Theme } from '../../theme';
 
-export const drawerContainerStyle = (expanded: boolean, isDesktop: boolean) => (
-  theme: Theme
-): SerializedStyles => css`
+export const drawerContainerStyle = (
+  expanded: boolean,
+  isDesktop: boolean,
+  isSmallDesktop: boolean
```

Comment:
Shouldn't we add for all the small devices instead of just isSmallDesktop?

Reply:
According to the designs, it's only for small desktop screens that we need to make the Side Navigation expandable on hover.